packages/api/src/feature_flag/validators.ts

```diff
@@ -17,3 +17,16 @@ export function validateCreateFeatureFlagRequest(
     throw validation.error;
   }
 }
+
+export function validateDeleteFeatureFlagParams(
```

Comment:
This is essentially the same as the param validator for update; perhaps they should be one function?